tests/py/test_ptq_trt_calibrator.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+import trtorch
+from trtorch.logging import *
+import torch
+import tensorrt as trt
+import torch.nn as nn
+from torch.nn import functional as F
+import torchvision
+import torchvision.transforms as transforms
+from trtorch.ptq import get_batch_size, get_batch, read_calibration_cache, write_calibration_cache
+from model_test_case import ModelTestCase
+
+
+class TestAccuracy(ModelTestCase):
+
+    def setUp(self):
+        self.input = torch.randn((1, 3, 32, 32)).to("cuda")
+        self.testing_dataset = torchvision.datasets.CIFAR10(root='./data',
+                                                            train=False,
+                                                            download=True,
+                                                            transform=transforms.Compose([
+                                                                transforms.ToTensor(),
+                                                                transforms.Normalize((0.4914, 0.4822, 0.4465),
+                                                                                     (0.2023, 0.1994, 0.2010))
+                                                            ]))
+
+        self.testing_dataloader = torch.utils.data.DataLoader(self.testing_dataset,
+                                                              batch_size=1,
+                                                              shuffle=False,
+                                                              num_workers=1)
+        # Test cases can assume using GPU id: 0
+        self.device = torch.device('cuda:0')
+        # Define attributes and member functions for the calibrator class
+        self.attribute_mapping = {
+            'data_loader': self.testing_dataloader,
+            'current_batch_idx': 0,
+            'batch_size': self.testing_dataloader.batch_size,
+            'dataset_iterator': iter(self.testing_dataloader),
+            'cache_file': None,
+            'device': self.device,
+            'use_cache': False,
+            'get_batch_size': get_batch_size,
+            'get_batch': get_batch,
+            'read_calibration_cache': read_calibration_cache,
+            'write_calibration_cache': write_calibration_cache
+        }
+
+        self.calibrator = type('DataLoaderCalibrator', (trt.IInt8EntropyCalibrator,), self.attribute_mapping)()

```

Comment:
I get that this is probably easier but we should at least try to implement the calibrator like they do in the TRT docs even if it just wraps the dataloadercalibrator functions. Like actually create a class that inherits properly and all